fetch_gazebo_demo/scripts/pick_place_demo.py

```diff
@@ -151,7 +151,7 @@ def updateScene(self):
             self.scene.addSolidPrimitive(obj.object.name,
                                          obj.object.primitives[0],
                                          obj.object.primitive_poses[0],
-                                         wait = False)
+                                         use_service = False)
```

Comment:
same comment as on #26 - I think you actually want to just remove the "wait = False" -- the new service interface is quite speedy and very reliable (unlike the old "wait for sync").

Reply:
I dont quite get you there. So you mean that its not necessary to indicate the 
**use_service = False**
? We just need to remove the wait?

And as for the addition of the fetchit_challenge package, everything ok there?

Reply:
You just need to remove the line, which will **use_service=True**
http://docs.ros.org/kinetic/api/moveit_python/html/classmoveit__python_1_1planning__scene__interface_1_1PlanningSceneInterface.html#a72a7f448769da906ae689420092ce4c4

The addition of the fetchit_challenge looks good enough for as the first pass, the diff is too large to really see in GitHub.